### PR TITLE
Accessibility feature

### PR DESCRIPTION
**Pull request by Esmatullah12**
Below are the things we did to make the portfolio more accessible:
1. Make the portfolio more accessible by giving some alternative information to the project images.
2. Give id to the navbar button to navigate that specific section in portfolio.
3. Transform the project images when it is hovered.
4. Change the navbar links color when it is hovered.

### CODE REVIEW

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Esmatullah12, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job making your site more accessible, well done 👏 👏 

## Required Changes ♻️
- According to the project requirements, If issues were found, they should link to the corresponding line of code in the Pull Request changes. Please fix this for all changes made 🙏
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Esmatullah</title>
+    <title>Esmatullah's Portfolio</title>
```
Comment (Gakiiberyl) [inline index.html:7]: :white_check_mark: Page title (We chose a title that adequately and briefly describes the contents of the page)

File: index.html [hunk 2]
```diff
@@ -18,9 +18,9 @@
                 <button class="nav-btn"><i class="fa-solid fa-bars fa-2x"></i></button>
             </div>
             <ul class="menue-btn primary-font">
-                <li><a href="">Portfolio</a></li>
-                <li><a href="">About</a></li>
-                <li><a href="">Contact</a></li>
+                <li><a href="index.html">Portfolio</a></li>
```
Comment (Gakiiberyl) [inline index.html:21]: :white_check_mark: We made the navbar links responsive

File: index.html [hunk 3]
```diff
@@ -43,7 +43,7 @@ <h1 >I’m Esmatullah <span><br></span> Glad to see you!</h1>
         <section id="works">
             <div class="projects">
                 <div>
-                    <img class="project-img project-left-img" src="images/first project.svg" alt="">
```
Comment (Gakiiberyl) [inline index.html:46]: :white_check_mark: Image text alternatives** (img alt="alt text") (We added appropiate alt texts to every image)

File: index.html [hunk 4]
```diff
@@ -70,7 +70,7 @@ <h2>Tonic</h2>
     
             <div class="projects">
                 <div class="project-img project-right-img">
-                    <img class="project-img" src="images/second project.svg" alt="">
```
Comment (Gakiiberyl) [inline index.html:73]: :white_check_mark: Image text alternatives** (img alt="alt text") (We added appropiate alt texts to every image)

**Comment by Esmatullah12** [comment]
Hi @mckent05 
I did not understand about your required changes that I should make it
Firstly, how can I write a report about the changes
Secondly, I did not understand about your second required change at all.

File: index.html [hunk 5]
```diff
@@ -178,15 +178,15 @@ <h1 class="about-title">About Myself</h1>
                         </div>
                         <ul class="languages">
                             <li class="programming-lang">
-                                <img src="images/javascrip.svg" alt="">
```
Comment (Gakiiberyl) [inline index.html:181]: :white_check_mark: Image text alternatives** (img alt="alt text") (We added appropiate alt texts to every image)

**Review by Gakiiberyl — COMMENTED** [review]
**We found the following during our Accessibility checks:**
:heavy_check_mark:Text headings (no issues found)

:heavy_check_mark:Color contrast (no issues found)

:heavy_check_mark:Resize (no issues found)

:heavy_check_mark:Moving content (no issues found)

:heavy_check_mark:Multimedia (no issues found because the project didn't require a video or audio of any kind)

:heavy_check_mark: The basic structure of the page (no issues found)

**Review by joel-kalema — APPROVED** [review]
Hi @Esmatullah12 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

- [ ] [OPTIONAL] Kindly, make sure every image has alt with appropriate alternative text, that's useful for interacting or understanding the page has an alt text 😃 

Capture


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**